### PR TITLE
Create Cavemancircus.com.xml

### PR DESCRIPTION
**Pull request by ryanbr**

### CODE REVIEW

File: src/chrome/content/rules/Cavemancircus.com.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<ruleset name="Cavemancircus.com">
+    <target host="cavemancircus.com" />
+    <target host="www.cavemancircus.com" />
+    <target host="webmail.cavemancircus.com" />
```
Comment (ivysrono) [inline src/chrome/content/rules/Cavemancircus.com.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/586109134#L323

**Comment by ivysrono** [comment]
@pipboy96 https://travis-ci.org/EFForg/https-everywhere/jobs/592969344#L330
Why?

**Comment by pipboy96** [comment]
https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/Cavemancirus.com.xml

**Comment by ivysrono** [comment]
😂

**Pull request closed without merging**